app/components/app-insights.js

```diff
@@ -33,4 +40,8 @@ function initAppInsights() {
     }
 }
 
-module.exports = initAppInsights;
+exports.trackTrace = function trackTrace(message,properties){
+    if(client){
+        client.trackTrace({message: message,properties: properties});
```

Comment:
Just curious, why `trackTrace` and not `trackEvent`, as trackEvent described as "Used to track user behavior or to monitor performance." and that sounds better fit than `trackTrace`.